core/state/statedb_utils.go

```diff
@@ -32,11 +36,41 @@ var (
 	}
 )
 
+// IsWhitelistedDeployer reads the contract storage to check if an address is allow to deploy
+func IsWhitelistedDeployerV2(statedb *StateDB, address common.Address, blockTime uint64) bool {
+	contract := common.HexToAddress(common.WhitelistDeployerSCV2)
+	whitelistAllSlot := slotWhitelistDeployerMappingV2[WHITELIST_ALL]
+	whitelistAll := statedb.GetState(contract, GetLocSimpleVariable(whitelistAllSlot))
+
+	if whitelistAll.Big().Cmp(big.NewInt(1)) == 0 {
```

Comment:
I think common.Big1 is better, it reuses that variable without initializing a new one